src/app/wallets/update-pending-invoices.ts

```diff
@@ -177,6 +177,11 @@ const updatePendingInvoice = async ({
       currency: DisplayCurrency.Usd,
     }
 
+    let amountDisplayCurrency = displayPaymentAmount.amount
```

Comment:
Is price generated dynamically from the amountDisplayCurrency and settlement amount?

Reply:
in this case is from price service.. line 159